backend/uclapi/roombookings/urls.py

```diff
@@ -6,4 +6,5 @@
     url(r'^rooms$', roombookings.views.get_rooms),
     url(r'^bookings$', roombookings.views.get_bookings),
     url(r'^equipment$', roombookings.views.get_equipment),
+    url(r'^free_rooms$', roombookings.views.free_rooms),
```

Comment:
I think this one should probably be `freeRooms` to align with our existing endpoints? What do you think @ChristopherHammond13

Reply:
@wilhelmklopp which current endpoint is camelCase? I think if any they're just internal APIs which I'm not so concerned about (as we can change / update those without any brekaing changes for clients)

Reply:
(I think we should adopt `underscored_endpoints` where possible as it's more Pythonic as per teh conversation about this in #134)

Reply:
Should we also name the views like `get_X_Y_Z_`?

Reply:
@jermenkoo I'd say so, since we'd do that anyway as it's Pythonic and PEP-8-y

Reply:
> @wilhelmklopp which current endpoint is camelCase?

I think you're right we're using flat case [(described here)](https://stackoverflow.com/questions/17326185/what-are-the-different-kinds-of-cases)

> (I think we should adopt underscored_endpoints where possible as it's more Pythonic as per teh conversation about this in #134)

disagree on this one because the endpoints aren't in python (talked about this in person today)
Our function/view names should definitely be pythonic